common/src/View/MapViewBase.cpp

```diff
@@ -1063,12 +1063,15 @@ namespace TrenchBroom {
             const std::vector<Model::LayerNode*> selectedObjectLayers = Model::findLayersUserSorted(nodes);
 
             QMenu* moveSelectionTo = menu.addMenu(tr("Move to Layer"));
+            bool anyMoveToLayerItemEnabled = false;
```

Comment:
Same here, maybe use `std::any_of`, it would even be faster because it stops iterating once a match is found.